Documentation/docs-site/content/contributing/_index.en.md

```diff
@@ -0,0 +1,6 @@
+---
+title: "Contributing to Fission"
+date: 2017-09-07T20:10:05-07:00
+draft: false
+weight: 50
+---
```

Comment:
As a starting point, you could just copy and paste the contributing.md that we have in the root